avahi-services/revpi-ssh.service

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone='no'?>
+<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
+<service-group>
+    <name replace-wildcards="yes">%h</name>
+
+    <service protocol="any">
+        <type>_ssh._tcp</type>
+        <port>22</port>
+	<txt-record>info=This is a RevolutionPi device!</txt-record>
```

This is mDNS, you never should use whitespaces here. This data filed will help us to identify the device in all ssh devices but this data will be send in each broadcast.
```suggestion
	<txt-record>kind=revolutionpi</txt-record>
```
would be enough.

As this are static values we should think about key and value @nbuchwitz 

What key makes sense and does not block an other system, which we could integrate in the future - maybe dynamic value generation like dev=connect / core ...